specification/proto-design-goals.md

```diff
@@ -0,0 +1,19 @@
+# Design Goals for OpenTelemetry Wire Protocol
+
+We want to design a telemetry data exchange protocol that has the following characteristics:
+
+- Be suitable for use between all of the following node types: instrumented applications, telemetry backends, local agents, stand-alone collectors/forwarders.
+
+- Have high reliability of data delivery and clear visibility when the data cannot be delivered.
```

Comment:
How do we think about de-duplicating the data? (e.g. in case of transmission failures, where sender has no idea whether the backend accepted the data or not)

Reply:
I'll have this in the RFC. In short, I suggest to aim for reliable delivery at the cost of possible duplicates. Details in the RFC.

Reply:
Thank you, I'll look into the RFC.